### PR TITLE
Incentive PR 4: claim Hard rewards via the Incentive module

### PR DESCRIPTION
**Pull request by denalimarsh**
- [x] Add message for hard rewards claims that allow users to claim rewards
- [x] Message claims all (supply, borrow, delegation) HARD rewards and deletes the claim object from the store
- [x] CLI/Rest API should also be updated for new transaction types
- [x] Added basic Hard claims queries

Questions:
- What should the Incentive module's API look like (Hard module vs. CDP module claims)?

### CODE REVIEW

File: x/incentive/keeper/payout.go [hunk 1]
```diff
@@ -47,7 +47,56 @@ func (k Keeper) ClaimReward(ctx sdk.Context, addr sdk.AccAddress, multiplierName
...
 	}
 
-	k.ZeroClaim(ctx, claim)
+	k.ZeroUSDXMintingClaim(ctx, claim)
+
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeClaim,
+			sdk.NewAttribute(types.AttributeKeyClaimedBy, addr.String()),
+			sdk.NewAttribute(types.AttributeKeyClaimAmount, claim.Reward.String()),
```
Comment (karzak) [inline x/incentive/keeper/payout.go:56]: Add claim type to the event

File: x/incentive/keeper/payout.go [hunk 2]
```diff
@@ -47,7 +47,56 @@ func (k Keeper) ClaimReward(ctx sdk.Context, addr sdk.AccAddress, multiplierName
...
+	}
+	rewardCoin := sdk.NewCoin(claim.Reward.Denom, rewardAmount)
+	length := ctx.BlockTime().AddDate(0, int(multiplier.MonthsLockup), 0).Unix() - ctx.BlockTime().Unix()
+
+	err := k.SendTimeLockedCoinsToAccount(ctx, types.IncentiveMacc, addr, sdk.NewCoins(rewardCoin), length)
+	if err != nil {
+		return err
+	}
+
+	k.ZeroHardLiquidityProviderClaim(ctx, claim)
```
Comment (karzak) [inline x/incentive/keeper/payout.go:99]: ditto add claim type to event

**Review by karzak — APPROVED** [review]
Tiny nits, otherwise LGTM.

File: x/incentive/keeper/payout.go (hunk 2 above)
Comment (denalimarsh) [reply x/incentive/keeper/payout.go:100]: https://github.com/Kava-Labs/kava/pull/780/commits/ac6d43f4968d631eb510ab7cd7fee7ae36bdfc7f

File: x/incentive/keeper/payout.go (hunk 1 above)
Comment (denalimarsh) [reply x/incentive/keeper/payout.go]: https://github.com/Kava-Labs/kava/pull/780/commits/ac6d43f4968d631eb510ab7cd7fee7ae36bdfc7f

**Pull request merged**